src/Reveal.Sdk.Dom.Tests/Data/DataSources/HostDataSourceFixture.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Reveal.Sdk.Dom.Data;
+using System;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class HostDataSourceFixture
+    {
+        [Fact]
+        public void GetHost_ReturnSameValue_WithSetValue()
+        {
+            // Arrange
+            var hostDataSource = new HostDataSource();
+            var expectedHost = "test-host";
+
+            // Act
+            hostDataSource.Host = expectedHost;
+
+            // Assert
+            Assert.Equal(expectedHost, hostDataSource.Host);
+        }
+
+        [Fact]  
+        public void GetPort_ReturnSameValue_WithSetPort()

```

Comment:
Can port be set with number?

Reply:
Updated

Reply:
Hi @hainv-ohio 
The port should be string instead of integer. Reference: https://stackoverflow.com/a/47992797